Makefile

```diff
@@ -5,15 +5,20 @@ tools:
 		npm i
 		go install github.com/ory/go-acc github.com/ory/x/tools/listx github.com/go-swagger/go-swagger/cmd/swagger github.com/go-bindata/go-bindata/go-bindata github.com/sqs/goreturns github.com/ory/sdk/swagutil
 
+# Runs full test suite including tests where databases are enabled
+.PHONY: test-legacy-migrations
+test-legacy-migrations:
+		make test-resetdb
+		source scripts/test-env.sh && go test -tags legacy_migration_test -failfast -timeout=20m ./internal/fizzmigrate
```

Comment:
isn't `make sqlbin` missing here?

Reply:
hm right I thought we don't need it as the old migrations should not change but if they change we should `make sqlbin` here